index.js

```diff
@@ -23,7 +27,7 @@ Babelify.prototype._transform = function (buf, enc, callback) {
 
 Babelify.prototype._flush = function (callback) {
   try {
-    var result = babel.transform(this._data, this._opts);
+    var result = this._babel.transform(this._data, this._opts)
```

Comment:
It'd be great to update this to use the callback version that landed in `beta.32`. e.g.
```
babel.transform(this._data, this._opts, (err, result) => {
```

Reply:
nice, but will need to check babel version before using the new API, I'll update it and run the tests.